boot.php

```diff
@@ -15,7 +23,12 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+if(file_exists(getcwd().'/'.$argv[1])){
+    $filename = getcwd().'/'.$argv[1];
+} else{
+    $filename = $argv[1];
```

Comment:
Passing paths like `~/roms/drmario.gb` not work if I remove the if/else statement